docs/config.md

```diff
@@ -226,6 +226,8 @@ net-max-open-requests = 100
 
 ```
 [cluster]
+# Unique name of the cluster.
```

Comment:
Might want to mention in the comment that this name is used to make sure that only metrictanks with the same cluster name connect to each other. Otherwise it's kind of ambiguous what this setting is for.